.swiftlint.yml

```diff
@@ -1,8 +1,12 @@
 included:
   - Sources
+excluded:
+  - Sources/MD5.swift
```

Comment:
`MD5.swift` is a dependency that will be removed in the future version of the SDK so we're excluding this file for now.

Reply:
These comments are best placed in the actual file.

```
excluded:
  - Sources/MD5.swift # this dependency will be removed in the future version of the SDK
```